pints/toy/_fitzhugh_nagumo_model.py

```diff
@@ -46,6 +72,12 @@ def n_outputs(self):
         return 2
 
     def simulate(self, parameters, times):
+        return self._simulate(parameters, times, False)
+
+    def simulate_with_sensitivities(self, parameters, times):
+        return self._simulate(parameters, times, True)
+
+    def _simulate(self, parameters, times, sensitivities):
         """ See :meth:`pints.ForwardModel.simulate()`. """
```

Comment:
This docstring is no longer appropriate! Should have a full one here with:

    Arguments:
        `parameters`
            Lalala

etc. that explains the extra sensitivities switch